cpp/src/layout/barnes_hut.hpp

```diff
@@ -52,9 +52,9 @@ void barnes_hut(raft::handle_t const &handle,
                 bool verbose                                  = false,
                 internals::GraphBasedDimRedCallback *callback = nullptr)
 {
-  cudaStream_t stream = handle.get_stream();
-  const edge_t e      = graph.number_of_edges;
-  const vertex_t n    = graph.number_of_vertices;
+  rmm::cuda_stream_view stream(handle.get_stream());
```

Comment:
Seems like we should use get_stream_view() for this, no? https://github.com/rapidsai/raft/blob/d5763480b899aa572afa92432fdf5bd6626a98a0/cpp/include/raft/handle.hpp#L102 
There are a couple of other occurrences in the PR